index.html

```diff
@@ -49,6 +49,232 @@ <h2>I’am a Software Developer</h2>
                 </section>
             </div>
         </div>
+        <div class="works">
```

Comment:
Kindly change this from a `div` tag to a `section ` tag. This is a stand alone section on its own and using semantic elements is important so that screen readers understand your project.